2_process/src/tally_site_obs.R

```diff
@@ -9,3 +9,7 @@ tally_site_obs <- function(site_data) {
     group_by(Site, State, Year) %>%
     summarize(NumObs = length(which(!is.na(Value))), .groups = "keep")
 }
+
+combine_obs_tallies <- function(...) {
+  bind_rows()
```

Comment:
I think you need this to be `bind_rows(...)` or it won't do anything? If it does, I am surprised by that behavior!